pkg/logs/docker/docker_test.go

```diff
@@ -26,6 +32,26 @@ func TestParseMessageShouldSucceedWithValidInput(t *testing.T) {
 	assert.Equal(t, []byte("anything"), dockerMsg.Content)
 }
 
+func TestParseMessageShouldHandleEmptyMessage(t *testing.T) {
+	msg, err := ParseMessage([]byte(header))
+	assert.Nil(t, err)
+	assert.Equal(t, 0, len(msg.Content))
+}
+
+func TestParseMessageShouldHandleTtyMessage(t *testing.T) {
+	msg, err := ParseMessage([]byte("2018-06-14T18:27:03.246999277Z foo"))
+	assert.Nil(t, err)
+	assert.Equal(t, "2018-06-14T18:27:03.246999277Z", msg.Timestamp)
+	assert.Equal(t, message.StatusInfo, msg.Status)
+	assert.Equal(t, []byte("foo"), msg.Content)
+}
+
+func TestParseMessageShouldHandleEmptyTtyMessage(t *testing.T) {
+	msg, err := ParseMessage([]byte("2018-06-14T18:27:03.246999277Z"))
```

Comment:
What about `2018-06-14T18:27:03.246999277Z ` ?

Reply:
Not sure what you mean, this is what Docker gives us when we tail a blank line from a tty container.